packages/memory/integration-tests-v5/src/output-processor-memory.test.ts

```
@@ -175,7 +175,9 @@ describe('Output Processor Memory Persistence Integration', () => {
     await new Promise(resolve => setTimeout(resolve, 100));
 
     // Retrieve messages from storage directly
-    const { messages: savedMessages } = await storage.listMessages({
+    const memoryStore = await storage.getStore('memory');
+    expect(memoryStore).toBeDefined();
+    const { messages: savedMessages } = await memoryStore?.listMessages({
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)